nginx/templates/includes/cors.conf.template

```diff
@@ -0,0 +1,5 @@
+more_set_headers -s 403 'Content-Type' '${CORS_CONTENT_TYPE}';
```

Comment:
The values should all be expanded from variables, I presume.

Reply:
Yes, but you're only replacing a single variable. You're not using the ones you've defined. And `CORS_CONTENT_TYPE` doesn't exist at all.

Reply:
WTH. Bad version. 🤦 

Don't know if I can write a decent test for this 🤔